graylog-storage-opensearch3/src/main/java/org/graylog/storage/opensearch3/indextemplates/OSSerializationUtils.java

```diff
@@ -52,8 +53,7 @@ public Map<String, Object> toMap(final PlainJsonSerializable openSearchSerializa
     public <T> T fromMap(final Map<String, Object> mapRepresentation,
                          final JsonpDeserializer<T> deserializer) throws JsonProcessingException {
         final String json = objectMapper.writeValueAsString(mapRepresentation);
```

Comment:
```suggestion
        StringWriter writer = new StringWriter();
        jsonpMapper.serialize(mapRepresentation, jsonpMapper.jsonProvider().createGenerator(writer));
        final String json = writer.toString();
```
If we use the JsonpMapper for serialization, we could get rid of the ObjectMapper and make the whole thing static. 
What do you think?

Reply:
Well, I highly dislike static methods. We could probably spawn a huge discussion why on a separate channel, if it feels important to you to turn it into static method...